rbac/management/seeds.py

```diff
@@ -26,11 +26,11 @@
 
 def on_complete(progress, tenant):
     """Explicitly close the connection for the thread."""
-    logger.info(f"Purging policy cache for tenant {tenant.tenant_name} [{progress}].")
+    logger.info(f"Purging policy cache for tenant {tenant} [{progress}].")
```

Comment:
HOw this log look like ? it dumps whole tenant object ?

Reply:
this returns the string representation of the tenant https://github.com/RedHatInsights/insights-rbac/blob/3756478b2b63dd7238fe73852d61030ece52eda2/rbac/api/models.py#L46 ... so in this case something like "Tenant 12345" ... but now I can see that better will be use `tenant.org_id` to be more readable